server/plugin/virtual_agent.go

```diff
@@ -256,12 +267,84 @@ func (p *Plugin) ProcessResponse(data []byte) error {
 			if _, err = p.DMWithAttachments(userID, p.CreateOutputCardAttachment(&data)); err != nil {
 				return err
 			}
+		case *OutputImage:
+			post, err := p.CreateOutputImagePost(res, userID)
+			if err != nil {
+				return err
+			}
+
+			if _, err = p.dm(userID, post); err != nil {
+				return err
+			}
 		}
 	}
 
 	return nil
 }
 
+func (p *Plugin) CreateOutputImagePost(body *OutputImage, userID string) (*model.Post, error) {
+	channel, appErr := p.API.GetDirectChannel(userID, p.botUserID)
+	if appErr != nil {
+		p.API.LogInfo("Couldn't get bot's DM channel", "user_id", userID, "error", appErr.Message)
+		return nil, appErr
+	}
+
+	resp, err := http.Get(body.Value)
+	if err != nil {
+		p.API.LogInfo("Error in getting file data from link", "error", err.Error())
```

Comment:
```suggestion
		p.API.LogInfo("Error in getting file data from link", "Error", err.Error())
```
Do this everywhere.
Why using `LogInfo` everywhere?